CHANGELOG.md

```diff
@@ -0,0 +1,60 @@
+<!--
+Copyright (C) Samuel Henrique <samueloph@debian.org>, Sergio Durigan
+Junior <sergiodj@debian.org> and many contributors, see the AUTHORS
+file.
+
+SPDX-License-Identifier: curl
+-->
+
+# Changelog
+
+## [UNRELEASED]
+ * Default to index.html as filename if non can be parsed from the URL.
+ * Percent-decode output filenames by default.
+ * New option to disable percent-decoding of output filenames: --no-decode-filename.
+ * wcurl.1: Fix typo on list of features.
+ * README/manpage: point to the curl issue tracker.
+ * README:
+   - Add a missing dash to the --dry-run command.
+   - Add a logo.
+   - Add brief section explaining about our testsuite.
+   - Remove HTML <a name> anchors.
+ * Symlink LICENSE to LICENSES/curl.txt.
+
+## [v2024.07.10]
+ * Change versioning to use dots as separators instead of dashes:
+     - Previous version: 2024-07-07
+     - New version: 2024.07.10
+ * Support older curl releases, minimum required version is now 7.46.0:
+     - Only set --no-clobber if curl is 7.83 or newer
+     - Only set --parallel if curl is 7.66 or newer
+ * Set --fail on curl, in order to return errors instead of saving as
+   output files.
+ * Add more tests.
+ * Remove the need for GNU coreutils' realpath for tests.
+ * wcurl.1: Update manpage with links to Github and Debian's Salsa.
+ * Update LICENSE file with new contributors.
+
+## [v2024-07-07]
+ * Drop getotp usage, non-linux environments are supported now.
```

Comment:
- `s/getotp/getopt/`

- Use backticks on `getopt`.

- I will personally use "GNU/Linux", but it's OK if you don't want to.  However, I believe we should at least capitalize the name "Linux" here.